aiven/datasource_project_vpc.go

```diff
@@ -29,11 +30,19 @@ func datasourceProjectVPCRead(ctx context.Context, d *schema.ResourceData, m int
 
 	for _, vpc := range vpcs {
 		if vpc.CloudName == cloudName {
+			foundVPC = true
 			d.SetId(buildResourceID(projectName, vpc.ProjectVPCID))
-			return diag.FromErr(copyVPCPropertiesFromAPIResponseToTerraform(d, vpc, projectName))
+			err = copyVPCPropertiesFromAPIResponseToTerraform(d, vpc, projectName)
+			if err != nil {
+				return diag.FromErr(err)
+			}
```

Comment:
Probably we can do even without `foundVPC` variable if we `return nil` here?

Reply:
Yeah sounds really better